### ISSUE TITLE
Question about circuit breaker behaviour with `no-op` encoding 

### ISSUE THREAD

**Issue by samstride** [krakend/krakend-circuitbreaker#12]
Hi,

@alombarte

Krakend version = v2.6.3.

The following config produces a log when the breaker's status changes:

```json
{
    "version": 3,
    "port": 8080,
    "echo_endpoint": true,
    "endpoints": [
        {
            "endpoint": "/test/{var}",
            "backend": [
                {
                    "host": [
                        "http://127.0.0.1:8080"
                    ],
                    "url_pattern": "/",
                    "extra_config": {
                        "qos/circuit-breaker": {
                            "interval": 60,
                            "timeout": 10,
                            "max_errors": 3,
                            "name": "krakend-circuitbreaker",
                            "log_status_change": true
                        }
                    }
                }
            ]
        }
    ]
}
```

However, when `no-op` encoding is added, no log is produced when the breaker's status changes:

```json
{
    "version": 3,
    "port": 8080,
    "echo_endpoint": true,
    "endpoints": [
        {
            "endpoint": "/test/{var}",
            "output_encoding": "no-op",
            "backend": [
                {
                    "encoding": "no-op",
                    "host": [
                        "http://127.0.0.1:8080"
                    ],
                    "url_pattern": "/",
                    "extra_config": {
                        "qos/circuit-breaker": {
                            "interval": 60,
                            "timeout": 10,
                            "max_errors": 3,
                            "name": "krakend-circuitbreaker",
                            "log_status_change": true
                        }
                    }
                }
            ]
        }
    ]
}
```

The [docs](https://www.krakend.io/docs/backends/circuit-breaker/#definition-of-failure) have this:

```
There is a small difference in behavior when you use the circuit breaker with no-op encoding vs. the rest of the encodings.
```

Is this behaviour expected?

Thanks.

**Comment by alombarte** [comment]
Hello @samstride,
The CB does not monitor status codes from responses when you use no-op, so it will act on other types of errors such as connectivity problems, timeouts, etc.

Try to put in your endpoint with no op a microsecond timeout, such as:
```
            "output_encoding": "no-op",
            "timeout": "1us",
```
Then you'll see how it is acting.

**Comment by stefan-geiger-mw** [comment]
Just a suggestion @alombarte 
It is possible to adapt the documentation https://www.krakend.io/docs/backends/circuit-breaker/
Maybe adding another line at the button to say that with no-op there is no check for http status
![image](https://github.com/krakend/krakend-circuitbreaker/assets/51077060/062b3654-37de-4c1c-bdf5-fe70988d6590)
Also this part of the documentation is kind of misleading, since the statement there is not true for `no-op`
![image](https://github.com/krakend/krakend-circuitbreaker/assets/51077060/00494dc5-a4c8-4d75-a110-b3bbda5d7be3)

**Comment by alombarte** [comment]
@stefan-geiger-mw  this will be addressed on the documentation of v2.7 when released
